HermesProxy/App.config

```diff
@@ -79,5 +79,11 @@
             Default:     "false"
     -->
     <add key="DebugOutput" value="false" />
+    <!--
+            Option:      PacketsLog
+            Description: Save each session's packets to a single file in PacketsLog directory.
+            Default:     "false"
+    -->
+    <add key="PacketsLog" value="false" />
```

Comment:
Should be on by default, so if something happens while i'm playing, i can check log and fix it. Otherwise i'm going to forget to enable it. We can still have the setting set to false in the binary release.